### ISSUE TITLE
Screenreader reads quick pick actions from next hovered item

### ISSUE THREAD

**Issue by misolori** [microsoft/vscode#94064]
Refs #93648

I'm not sure if this is expected behavior but this felt odd to me.

![image](https://user-images.githubusercontent.com/35271042/78076579-8cd1cf80-735b-11ea-9e96-22e09f7c63f8.png)

1. Open quick pick
2. Navigate down the list
3. Place your cursor below the active line
4. Tab through the actions
5. :bug: the screen reader reads the next line action items. if you didn't see that your cursor was hovering over another line you'd no idea why the actions kept repeating.

I would at least expect for the screen reader to start reading the next line and then go directly to the button action text.

**Comment by bpasero** [comment]
@misolori wondering if that is a generic list issue, can you reproduce this from a stable view such as SCM?

![image](https://user-images.githubusercontent.com/900690/78110998-41233280-73fc-11ea-9285-cf94537126e2.png)

**Comment by misolori** [comment]
You're right that it's likely a list issue, the same happens in other list views (like scm).

**Comment by bpasero** [comment]
cc @isidorn

**Comment by isidorn** [comment]
Yeah, agree this is an issue with action bar and showing on hover.
Basically action bar that is showed on hover should not have a tabindex. The only actionbar that should be focused on is the one where the list focus is. Not the one that is shown on hover.
I belive this is not a list issue, but an issue of all the clients that use list + actionbar.

**Comment by joaomoreno** [comment]
I like that solution @isidorn, wanna take it?

**Comment by isidorn** [comment]
@joaomoreno yeah.

**Comment by isidorn** [comment]
Looked into this and the solution is not so trivial due to the fact that `tabindex` can not be set via CSS but only in javascript. And currently every client of the actionbar controls its visibility using css.
The solution would be that every clients renderer sets the tabindex of the actionbar to -1 but sets it back to 0 when the focus is set on the element. This is a bit ugly since the renderers do not have a pointer to the list.

Also blind users mostly do not use the mouse and thus this situation will not happen that often, that an unrelated element has hover.

Due to the above I am pushing this issue out to backlog.

**Comment by isidorn** [comment]
Due to my above explanation closing this as will not fix.
